Library/Formula/kubernetes-helm.rb

```diff
@@ -0,0 +1,15 @@
+class KubernetesHelm < Formula
+  desc "The package manager for Kubernetes"
+  homepage "https://helm.sh"
+  url "https://dl.bintray.com/deis/helm/helm-0.5.0+1689ee4-darwin-amd64.zip"
+  version "0.5.0"
+  sha256 "901b6d5dcb12af1a351906ba8eda2c5c5860028aa58a64e0ade8befd1967a318"
+
+  def install
+    bin.install "helm"
+  end
+
+  test do
+    assert_match /^helm version 0.5.0/, shell_output("#{bin}/helm -v 2>&1", 0)
```

Comment:
Can’t we write a test that does something more substantial than e.g. `-h` or `-v`?